strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/MavenGroupRepositoryProviderTest.java

```diff
@@ -554,72 +420,43 @@ public void testGroupAgainstNestedRepository(TestInfo testInfo)
     }
 
     @Test
-    public void testGroupExcludes(TestInfo testInfo)
+    @ExtendWith({RepositoryManagementTestExecutionListener.class, ArtifactManagementTestExecutionListener.class})
+    public void testGroupExcludes(@MavenRepository(repository = "grpt-releases-tge-1") Repository releases1,
+                                  @MavenRepository(repository = "grpt-releases-tge-2") Repository releases2,
+                                  @TestRepository.Group(repositories = { "grpt-releases-tge-1",
+                                                                         "grpt-releases-tge-2" }, 
+                                                        rules = { @Rule(repositories = { "grpt-releases-tge-1" },
+                                                                        pattern = ".*(com|org)/artifacts.denied.*",
+                                                                        type = RoutingRuleTypeEnum.DENY)})
```

Comment:
@fuss86 : Have a look at how simple stuff can now be in tests! Good job, @sbespalov ! :)
This will really save us from having to deal with all the stupid boiler-plate code.